src/main/java/PostgresCommunicator.java

```diff
@@ -11,7 +11,7 @@ public PostgresCommunicator(String databaseURL) {
     }
 
     public void writeToDatabase(Opportunity opportunity) throws SQLException, ClassNotFoundException {
-        String sqlQuery = String.format("INSERT INTO OPPORTUNITIES (name, description, proposed_cost, user_name, stage, opportunity_uuid) VALUES ('%s', '%s', %d, '%s', '%s', '%s');", opportunity.getName(), opportunity.getDescription(), opportunity.getProposedCost(), opportunity.getUserName(), opportunity.getStage(), opportunity.getUuid());
+        String sqlQuery = String.format("INSERT INTO OPPORTUNITIES (name, description, proposed_cost, user_name, stage, opportunity_uuid, opportunity_date) VALUES ('%s', '%s', %d, '%s', '%s', '%s', TO_DATE('%s', 'dd/mm/yyyy'));", opportunity.getName(), opportunity.getDescription(), opportunity.getProposedCost(), opportunity.getUserName(), opportunity.getStage(), opportunity.getUuid(), opportunity.getDate());
```

Comment:
is the to_date necessary? I don't think it needs to—I think YYYY-MM-DD works?

Reply:
Looks like it is needed as without I get this error message: `org.postgresql.util.PSQLException: ERROR: column "opportunity_date" is of type date but expression is of type record`

Reply:
Ah